app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -421,10 +421,14 @@ function AcuantCapture(
   }
 
   function onAcuantImageCaptureSuccess(nextCapture: AcuantSuccessResponse) {
-    const { image, cardtype, dpi, moire, glare, sharpness } = nextCapture;
+    let { image, cardType, dpi, moire, glare, sharpness } = nextCapture;
```

Comment:
What do you think about keeping all const except for cardType? Less clean but will ensure those values don't get changed.

Reply:
I agree with this point, if cardType is getting special handling I think it should be separate.

Reply:
I ended up refactoring this in a way that allowed `cardType` to be a constant, but I do agree with your point here. 👍🏻